packages/react-components/react-charts-preview/library/src/components/CommonComponents/Popover.types.ts

```diff
@@ -20,18 +20,21 @@ export interface IPopoverComponentProps {
   YValueHover?: IYValueHover[];
   descriptionMessage?: string;
   ratio?: [number, number];
+  isCartesian?: boolean;
 }
 
 export interface IPopoverComponentStyles {
   calloutContentRoot: string;
   calloutDateTimeContainer: string;
   calloutContentX: string;
-  calloutBlockContainer: string;
+  calloutBlockContainerCartesian: string;
+  calloutBlockContainerNonCartesian: string;
```

Comment:
the cartesian and non cartesian style classes should be internal to the style and not exposed via the interface. They should be conditionally merged based on the prop passed in.
See the usage of chartTitleLeft4pMargin in horizontal bar chart

Reply:
Updated